vyper/types.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, members):
         self.members = copy.copy(members)
 
     def __eq__(self, other):
-        return other.__class__ == StructType and other.members == self.members
```

Comment:
What was behind this change?

Reply:
This was clearly a copy paste mistake, there is no way you'd want equalise tuple to struct, especially because structtype looks different internally ;)